core/src/main/java/hudson/model/Queue.java

```diff
@@ -343,6 +343,14 @@ public void setSorter(QueueSorter sorter) {
     }
 
     /**
+     * Simple queue state persistence object.
+     */
+    public static class State {
```

Comment:
Nope, XStream deals with `private` types and fields, just like Java serialization.